### ISSUE TITLE
Possibly wrong CacheType?

### ISSUE THREAD

**Issue by OmgImAlexis** [sebhildebrandt/systeminformation#618]
If I include `Level` I get this back from the query. Is this meant to be checking  `CheckType` and not `Level`?

```powershell
Get-WmiObject Win32_CacheMemory | select Level,CacheType,InstalledSize,Purpose | ConvertTo-Json
```

```js
[
  { Level: 3, CacheType: 5, InstalledSize: 384, Purpose: 'L1 Cache' },
  { Level: 4, CacheType: 5, InstalledSize: 1536, Purpose: 'L2 Cache' },
  { Level: 5, CacheType: 5, InstalledSize: 9216, Purpose: 'L3 Cache' }
]
```

https://github.com/sebhildebrandt/systeminformation/blob/e1fafcc3404ec59261313d48104fb43ae6771c93/lib/cpu.js#L857-L864

**Comment by sebhildebrandt** [comment]
@OmgImAlexis Just to be sure ... what  is the output of 

```
Get-WmiObject Win32_processor | fl *
```

on this machine? And (if this is not clear from the output above) what processor you have in your machine?
Thank you in advance

**Comment by OmgImAlexis** [comment]
```
PSComputerName                          : ALEXIS-TOWER
Availability                            : 3
CpuStatus                               : 1
CurrentVoltage                          : 9
DeviceID                                : CPU0
ErrorCleared                            :
ErrorDescription                        :
LastErrorCode                           :
LoadPercentage                          : 3
Status                                  : OK
StatusInfo                              : 3
AddressWidth                            : 64
DataWidth                               : 64
ExtClock                                : 100
L2CacheSize                             : 1536
L2CacheSpeed                            :
MaxClockSpeed                           : 2808
PowerManagementSupported                : False
ProcessorType                           : 3
Revision                                :
SocketDesignation                       : CPUSocket
Version                                 :
VoltageCaps                             :
__GENUS                                 : 2
__CLASS                                 : Win32_Processor
__SUPERCLASS                            : CIM_Processor
__DYNASTY                               : CIM_ManagedSystemElement
__RELPATH                               : Win32_Processor.DeviceID="CPU0"
__PROPERTY_COUNT                        : 57
__DERIVATION                            : {CIM_Processor, CIM_LogicalDevice, CIM_LogicalElement,
                                          CIM_ManagedSystemElement}
__SERVER                                : ALEXIS-TOWER
__NAMESPACE                             : root\cimv2
__PATH                                  : \\ALEXIS-TOWER\root\cimv2:Win32_Processor.DeviceID="CPU0"
Architecture                            : 9
AssetTag                                : To Be Filled By O.E.M.
Caption                                 : Intel64 Family 6 Model 158 Stepping 10
Characteristics                         : 236
ConfigManagerErrorCode                  :
ConfigManagerUserConfig                 :
CreationClassName                       : Win32_Processor
CurrentClockSpeed                       : 2808
Description                             : Intel64 Family 6 Model 158 Stepping 10
Family                                  : 205
InstallDate                             :
L3CacheSize                             : 9216
L3CacheSpeed                            : 0
Level                                   : 6
Manufacturer                            : GenuineIntel
Name                                    : Intel(R) Core(TM) i5-8400 CPU @ 2.80GHz
NumberOfCores                           : 6
NumberOfEnabledCore                     : 6
NumberOfLogicalProcessors               : 6
OtherFamilyDescription                  :
PartNumber                              : To Be Filled By O.E.M.
PNPDeviceID                             :
PowerManagementCapabilities             :
ProcessorId                             : BFEBFBFF000906EA
Role                                    : CPU
SecondLevelAddressTranslationExtensions : False
SerialNumber                            : To Be Filled By O.E.M.
Stepping                                :
SystemCreationClassName                 : Win32_ComputerSystem
SystemName                              : ALEXIS-TOWER
ThreadCount                             : 6
UniqueId                                :
UpgradeMethod                           : 1
VirtualizationFirmwareEnabled           : False
VMMonitorModeExtensions                 : False
Scope                                   : System.Management.ManagementScope
Path                                    : \\ALEXIS-TOWER\root\cimv2:Win32_Processor.DeviceID="CPU0"
Options                                 : System.Management.ObjectGetOptions
ClassPath                               : \\ALEXIS-TOWER\root\cimv2:Win32_Processor
Properties                              : {AddressWidth, Architecture, AssetTag, Availability...}
SystemProperties                        : {__GENUS, __CLASS, __SUPERCLASS, __DYNASTY...}
Qualifiers                              : {dynamic, Locale, provider, UUID}
Site                                    :
Container                               :
```

**Comment by sebhildebrandt** [comment]
@OmgImAlexis will release my fix after having it tested on all my windows machines

**Comment by sebhildebrandt** [comment]
Thank you @OmgImAlexis, I checked it again on different machines and also Win32_CacheMemory class documentation:

CacheType - is type of cache, where 
- Other (1)
- Unknown (2)
- Instruction (3)
- Data (4)
- Unified (5)
...

Checking the type of l1 cache (cacheType) - l1i (L1 instruction) and l1d (L1 data), as I did in my code should be ok. In your case, as there is no seperate instruction / data L1 cache, I checked the cpu specs. In such cases L1i and L1d are specified as half of overall l1 cache. This part was missing. Added it already. V5.9.13 just released.
